dev/com.ibm.ws.security.javaeesec.cdi/resources/com/ibm/ws/security/javaeesec/cdi/internal/resources/JavaEESecMessages.nlsprops

```diff
@@ -17,50 +17,45 @@
+JAVAEESEC_CDI_ERROR_LOGIN_CONFIG_EXISTS=CWWKS1931E: The {1} application  failed deployment because both a login-config element in the web.xml file and the HttpAuthenticationsMechanism element were specified for the {0} module. Make sure that only one authentication  mechanism is configured.
+JAVAEESEC_CDI_ERROR_LOGIN_CONFIG_EXISTS.explanation=The application deployment failed because both the login-config element and the HttpAuthenticationMechanism element are defined. Only one of these authentication mechanisms can be used in your application.
+JAVAEESEC_CDI_ERROR_LOGIN_CONFIG_EXISTS.useraction=Make sure that only one of  the authentication mechanisms is configured.
 
-JAVAEESEC_CDI_ERROR_LOGIN_CONFIG_EXISTS=CWWKS1931E: The application deployment failed because a login-config element was found in the web.xml file for the {0} module in the {1} application. To enable the HttpAuthenticationMechanism bean, remove the login-config element from the web.xml file.
-JAVAEESEC_CDI_ERROR_LOGIN_CONFIG_EXISTS.explanation=The application deployment failed because both the login-config element and the HttpAuthenticationMechanism bean are defined. Only one of these authentication mechanisms can be used in your application.
-JAVAEESEC_CDI_ERROR_LOGIN_CONFIG_EXISTS.useraction=To enable the HttpAuthenticationMechanism bean, remove the login-config element from the web.xml file.
-
-JAVAEESEC_CDI_WARNING_NO_REALM_NAME=CWWKS1932W: The realm name for basic authentication is not set, so the default value, defaultRealm, was used.
-JAVAEESEC_CDI_WARNING_NO_REALM_NAME.explanation=Because realm name for the basic authentication is not defined, the realm name was set to the default value of defaultRealm.
-JAVAEESEC_CDI_WARNING_NO_REALM_NAME.useraction=To set the realm name, define it on a BasicAuthenticationMechanismDefinition annotation in the application. Alternatively, when the overrideHttpAuthenticationMechanism attribute is set as BASIC, set the value of attribute basicAuthenticationMechanismRealmName.
-
-JAVAEESEC_CDI_ERROR_PASSWORDHASH_INVALID_PARAM=CWWKS1933E: The value {0} of the configuration parameter {1} is invalid.
-JAVAEESEC_CDI_ERROR_PASSWORDHASH_INVALID_PARAM.explanation=The value of the configuration parameter is not valid.
+JAVAEESEC_CDI_ERROR_PASSWORDHASH_INVALID_PARAM=CWWKS1933E: The password hash for the DatabaseIdentityStore attribute is not valid because the {1} configuration parameter has a value of {0} that is not valid.
+JAVAEESEC_CDI_ERROR_PASSWORDHASH_INVALID_PARAM.explanation=The password hash for the DatabaseIdentityStore attribute is not valid because the value of the configuration parameter is not valid. <=== Explain what values are not valid, or given an example of what is not valid, or tell them what are common values that occur that are not valid, or tell them what is valid, etc.
 JAVAEESEC_CDI_ERROR_PASSWORDHASH_INVALID_PARAM.useraction=Make sure that the configuration parameter is set correctly.
```

Comment:
I think there might be an error in this message starting with **<===**. Maybe some content is missing?

+JAVAEESEC_CDI_ERROR_PASSWORDHASH_INVALID_PARAM.explanation=The password hash for the DatabaseIdentityStore attribute is not valid because the value of the configuration parameter is not valid. <=== Explain what values are not valid, or given an example of what is not valid, or tell them what are common values that occur that are not valid, or tell them what is valid, etc.